app/mailers/candidate_mailer.rb

```diff
@@ -402,12 +402,12 @@ def email_for_candidate(application_form, args = {})
   end
 
   def candidate_magic_link(candidate)
-    raw_token = candidate.refresh_magic_link_token!
+    raw_token = candidate.create_magic_link_token!
```

Comment:
I still don't like creating this in the mailer but last time we concluded we didn't want it floating around as an arg in Sidekiq. Could we mitigate that?

Reply:
I think we need to rethink that because it's in the email log now anyway, let's look at that next year.

Card: https://trello.com/c/GDGIVX65/399-draft-revisit-assumptions-about-security